domain-proxy/go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/redhat-appstudio/jvm-build-service/domain-proxy
+
+go 1.23
```

Comment:
Is there a reason this is using a different version of golang to the top level (although there is https://github.com/redhat-appstudio/jvm-build-service/pull/2045 I guess )

Reply:
The reason is to use the latest. It's a separate application to the 'top level'. We could update the 'top level' too I suppose.